examples/AdminClient/Program.cs

```diff
@@ -84,7 +95,218 @@ static async Task CreateTopicAsync(string bootstrapServers, string topicName)
+                    }
+                });
+            }
+            return aclBindings;
+        }
+
+
+        static List<AclBindingFilter> ParseAclBindingFilters(string[] args)
+        {
+            var aclBindings = ParseAclBindings(args);
+            return aclBindings.Select(aclBinding => aclBinding.ToFilter()).ToList();
+        }
+
+
+        static void PrintAclBindings(List<AclBinding> aclBindings)
+        {
+            foreach (AclBinding aclBinding in aclBindings)
+            {
+                Console.WriteLine($"\t{aclBinding}");
+            }
+        }
+
+        static async Task CreateAclsAsync(string bootstrapServers, string[] commandArgs)
+        {
+            var numCommandArgs = commandArgs.Length;
+            var printUsage = numCommandArgs == 0 || numCommandArgs % 7 != 0;
+            List<AclBinding> aclBindings = null;
+            if (!printUsage)
+            {
+                aclBindings = ParseAclBindings(commandArgs);
+            }
+            printUsage = aclBindings == null;
+
+            if (printUsage)
+            {
+                Console.WriteLine("usage: .. <bootstrapServers> create-acls <resource_type1> <resource_name1> <resource_patter_type1> "+
+                "<principal1> <host1> <operation1> <permission_type1> ..");
+                Environment.ExitCode = 1;
+                return;
+            }
+
+            using (var adminClient = new AdminClientBuilder(new AdminClientConfig { BootstrapServers = bootstrapServers }).Build())
+            {
+                try
+                {
+                    var results = await adminClient.CreateAclsAsync(aclBindings);
+                    Console.WriteLine("All the create ACL operations completed successfully");
+                }
+                catch (CreateAclsException e)
+                {
+                    Console.WriteLine("Some ACL operations failed");
+                    int i = 0;
+                    foreach (var result in e.Results)
+                    {
+                        if (!result.Error.IsError)
+                        {
+                            Console.WriteLine($"Create ACLs operation {i} completed successfully");
+                        }
+                        else
+                        {
+                            Console.WriteLine($"An error occurred in create ACL operation {i}: Code: {result.Error.Code}" +
+                            $", Reason: {result.Error.Reason}");
+                        }
+                        ++i;
+                    }
+                }
+                catch (KafkaException e)
+                {
+                    Console.WriteLine($"An error occurred calling the CreateAcls operation: {e.Message}");
+                }
+            }
+        }
+
+        static async Task DescribeAclsAsync(string bootstrapServers, string[] commandArgs)
+        {
+            var numCommandArgs = commandArgs.Length;
+            var printUsage = numCommandArgs != 7;
```

Comment:
same comment as before, rearrange to use exceptions + the nargs check taken care of elsewhere.